lib/server.go

```diff
@@ -491,6 +497,33 @@ func NewServer(
 		_minFeeRateNanosPerKB, _blockCypherAPIKey, _runReadOnlyUtxoViewUpdater, _dataDir,
 		_mempoolDumpDir, false)
 
+	// TODO: Initialize PoSMempool properly
+	globalParams := DbGetGlobalParamsEntry(_chain.DB(), _snapshot)
+	// TODO: how do we actually want to construct this?
+	readOnlyLatestBlockView, err := NewUtxoView(_chain.DB(), _params, nil, _snapshot, _chain.eventManager)
+	_posMempool := NewPosMempool(
+		_params, globalParams, readOnlyLatestBlockView,
+		uint64(_chain.blockTip().Height),
+		_mempoolDumpDir+"pos", // TODO: separate arg for mempool dump dir for pos mempool?
+		false,
+		1e9, // TODO: replace with real value for maxMempoolPosSizeBytes
+		1e4, // TODO: replace with real value for mempoolBackupIntervalMillis
+	)
+	if err = _posMempool.Start(); err != nil {
+		return nil, errors.Wrapf(err, "NewServer: Problem starting PoS mempool"), true
+	}
+
+	// TODO: how do we want to handle a situation where we don't have a tip block?
+	_feeEstimator := &PoSFeeEstimator{}
```

Comment:
Also, can you create a constructor for the fee estimator so we're not manually instantiating the struct everywhere? This way, if we ever need to do extra work during construction, we have a constructor available to do that work in
```suggestion
	// TODO: how do we want to handle a situation where we don't have a tip block?
	_feeEstimator := NewPoSFeeEStimator()
```